config/initializers/subsites.rb

```diff
@@ -1 +1,34 @@
-SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env]
\ No newline at end of file
+SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env].with_indifferent_access
+SUBSITES['sites'] = {}.with_indifferent_access
```

Comment:
used in the sites_controller to identify the publisher of Concepts for the index action.